tests/tt_eager/python_api_testing/unit_testing/backward_ops/test_backward_binary_eq.py

```diff
@@ -38,7 +39,8 @@ def test_bw_binary_eq(input_shapes, device):
         (torch.Size([1, 3, 320, 384])),
     ),
 )
-@pytest.mark.parametrize("are_required_outputs", [[True, True], [True, False], [False, True]])
+# @pytest.mark.parametrize("are_required_outputs", [[True, True], [True, False], [False, True]])
```

Comment:
Why do we reduce the test coverage?

Reply:
Currently we have not provided optional tensor support @ayerofieiev-tt . Hence, the output tensors cannot be null. But we have raised a separate PR for it. Once that gets merged, we can make the test file work for all cases